speech.py

```diff
@@ -63,48 +96,126 @@ def connect_wave(self, cb):
     def connect_idle(self, cb):
         self._cb['idle'] = self.connect('idle', cb)
 
+    def set_kokoro_voice(self, voice_name): #TODO - function set here to this can be called for when switching happens via GUI
+        if voice_name in self.kokoro_voices:
+            self.current_kokoro_voice = voice_name
+            logger.debug(f"Kokoro voice set to: {voice_name}")
+        else:
+            logger.warning(f"Invalid Kokoro voice: {voice_name}.")
+
+    def get_available_kokoro_voices(self): #TODO - function set here to this can be called for when retreiving happens via GUI
```

Comment:
Another confusing comment, this method doesn't seem to be used anywhere, if that's the case then we don't need it.

Reply:
Yes, saw this after I looked further.